examples/pytorch_lightning_simple.py

```diff
@@ -165,7 +165,15 @@ def objective(trial):
 
 
 if __name__ == '__main__':
-    study = optuna.create_study(direction='maximize')
+    parser = argparse.ArgumentParser(description='PyTorch Lightning example.')
+    parser.add_argument('--pruning', '-p', action='store_true',
+                        help='Activate the pruning feature. `MedianPruner` stops unpromising '
+                             'trials at the early stages of training.')
+    args = parser.parse_args()
+    args.pruning = True
```

Comment:
I think this line should be removed because this seems to always enable pruning.
```suggestion
```

Reply:
I forgot to delete the line used for debugging, so I fixed it.